doc/guides/bluetooth/autopts/autopts-linux.rst

```diff
@@ -344,12 +355,40 @@ Server and client by default will run on localhost address. Run server:
    :width: 700
    :align: center
 
-Run client:
+Testing Zephyr Host Stack on QEMU:
+
+.. code-block::
+
+    # Attach Bluetooth Controller(with H4 is an example)
+    # Note:Baudrate 115200 for bbc microbit boards
+    sudo btattach -B /dev/ttyXXX -S 1000000 -R
+
+    sudo bluez/tools/btproxy -u -z
+
+    python ./autoptsclient-zephyr.py "C:\Users\USER_NAME\Documents\Profile Tuning Suite\PTS_PROJECT\PTS_PROJECT.pqw6"
+    ~/zephyrproject/build/zephyr/zephyr.elf -i SERVER_IP -l LOCAL_IP
+
+
+Testing Zephyr Host Stack on native posix:
+
+.. code-block::
+
+    # Attach Bluetooth Controller(with H4 is an example)
+    # Note:Baudrate 115200 for bbc microbit boards
+    sudo btattach -B /dev/ttyXXX -S 1000000 -R
+
+    west build -b native_posix zephyr/tests/bluetooth/tester/ -DOVERLAY_CONFIG=overlay-native.conf
+
+    sudo python ./autoptsclient-zephyr.py "C:\Users\USER_NAME\Documents\Profile Tuning Suite\PTS_PROJECT\PTS_PROJECT.pqw6"
+    ~/zephyrproject/build/zephyr/zephyr.exe -i SERVER_IP -l LOCAL_IP --hci 0
```

Comment:
Does this actually work for Native Posix? 

Usually when I run with Native Posix I don't use `btattach`. 

the `--hci 0` is also different when I usually run, as my argument is `-bt-dev=0`, as well as needing to use the `-rt` argument to make it run more proper.

Reply:
see https://github.com/intel/auto-pts/pull/466

Reply:
In fact, this and the above “sudo python .......” are a whole command, but it seems to be separate because of the indentation problem.

Reply:
Oooooh, the `~/zephyrproject/build/zephyr/zephyr.exe` is an argument to `autoptsclient-zephyr.py`? That makes so much more sense. I think the documentation could benefit from either keeping it as a single line, or by adding a `\` that properly splits a single line into multiple lines.